graylog2-web-interface/src/components/graylog/Tooltip.jsx

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line no-restricted-imports
+import { Tooltip as BootstrapTooltip } from 'react-bootstrap';
+import styled from 'styled-components';
+
+import teinte from 'theme/teinte';
+
+const Tooltip = styled(BootstrapTooltip)`
+  &.top .tooltip-arrow {
+    bottom: 0;
+    border-top-color: ${teinte.primary.tre};
+  }
+  &.top-left .tooltip-arrow {
+    border-top-color: ${teinte.primary.tre};
+  }
+  &.top-right .tooltip-arrow {
+    border-top-color: ${teinte.primary.tre};
+  }
+  &.right .tooltip-arrow {
+    border-right-color: ${teinte.primary.tre};
+  }
+  &.left .tooltip-arrow {
+    border-left-color: ${teinte.primary.tre};
+  }
+  &.bottom .tooltip-arrow {
+    border-bottom-color: ${teinte.primary.tre};
+  }
+  &.bottom-left .tooltip-arrow {
+    border-bottom-color: ${teinte.primary.tre};
+  }
+  &.bottom-right .tooltip-arrow {
+    border-bottom-color: ${teinte.primary.tre};
+  }
```

Comment:
Is there a reason, you are are you not chaining the css selectors? Like modularity or readability?

```
  &.top .tooltip-arrow {
    bottom: 0;
  }
  &.bottom .tooltip-arrow, &.bottom-left .tooltip-arrow,  &.bottom-right .tooltip-arrow {
    border-bottom-color: ${teinte.primary.tre};
  }
  &.top-left .tooltip-arrow,  &.top-right .tooltip-arrow, &.top .tooltip-arrow {
    border-top-color: ${teinte.primary.tre};
  }
  &.right .tooltip-arrow {
    border-right-color: ${teinte.primary.tre};
  }
  &.left .tooltip-arrow {
    border-left-color: ${teinte.primary.tre};
  }
```